plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceRegistry.java

```diff
@@ -83,6 +83,7 @@ public class DataSourceRegistry<T extends DataSourceDescriptor> implements DBPDa
     private final DBVModel.ModelChangeListener modelChangeListener = new DBVModel.ModelChangeListener();
     private volatile ConfigSaver configSaver;
     private DBACredentialsProvider authCredentialsProvider;
+    private String navigatorView;
```

Comment:
Is it persisted somehow?

Reply:
It is not used anywhere.
Project-specific navigator settings are not stored anywhere in project.